Gemfile

```diff
@@ -29,6 +31,8 @@ gem 'spring',        group: :development
 
 # Use ActiveModel has_secure_password
 gem 'bcrypt', '~> 3.1.7'
+gem 'rails_12factor'
+gem 'faker'
```

Comment:
I assume you're using `faker` in development and maybe test. But right now it's available in all environments. Make sure you're explicit about which environments `faker` is in.